tests/detect-ftp/ftp-command-01/test.yaml

```diff
@@ -0,0 +1,49 @@
+pcap: ../../bug-3519/input.pcap
+
+requires:
+  version: 8
+
+
+checks:
+  - filter:
+      count: 1
+      match:
+        event_type: alert
+        ftp.command: USER
```

Comment:
can you add the `alert.signature_id` checks here as well? Want to make sure the correct sig matches in each case